### PR TITLE
Feature/straglr

### PR DESCRIPTION
**Pull request by creisle**
New Features
- Add straglr converter (https://github.com/bcgsc/mavis/issues/317)

### CODE REVIEW

**Review by zhemingfan — DISMISSED** [review]

File: docs/inputs/sv_callers.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+| [CuteSV (Jiang, 2020)](../../background/citations#jiang-2020)               | `1.0.10`          | `*.vcf`                                       |
+| [DeFuse (McPherson. 2011)](../../background/citations#mcpherson-2011)       | `0.6.2`           | `results/results.classify.tsv`                |
+| [DELLY (Rausch, 2012)](../../background/citations#rausch-2012)              | `0.6.1` `0.7.3`   | `combined.vcf` (converted from bcf)           |
+| [Manta (Chen, 2016)](../../background/citations#chen-2016)                  | `1.0.0`           | `{diploidSV,somaticSV}.vcf`                   |
+| [Pindel (Ye, 2009)](../../background/citations#ye-2009)                     | `0.2.5b9`         | `Tools main output file(s)`                   |
+| [Sniffles (Sedlazeck, 2018)](../../background/citations#sedlazeck-2018)     | `1.0.12b`         | `*.vcf`                                       |
+| [STAR-Fusion (Haas, 2017)](../../background/citations#haas-2017)            | `1.4.0`           | `star-fusion.fusion_predictions.abridged.tsv` |
+| [Straglr (Chiu, 2021)](../../background/citations#chiu-2021)                |                   |
```
Comment (mattdoug604) [inline docs/inputs/sv_callers.md:44]: Is there a missing `|` at the end of this line? Also no version or file information?

Comment (creisle) [reply docs/inputs/sv_callers.md:44]: Yup, though oddly it doesn't seem to affect the render at all. As far as I can tell this tool isn't versioned (see github)

Comment (zhemingfan) [reply docs/inputs/sv_callers.md]: I've asked Readman if there's a [stable release version](https://github.com/bcgsc/straglr/issues/8), we shouldn't be using a non-versioned iteration of this tool.

Comment (creisle) [reply docs/inputs/sv_callers.md]: oic, I think I didn't look at the releases in enough detail. I just saw it was called "final release before publication" and didn't realize the tag was semanticly versioned still (i've never seen someone do that before). I've pinged the user on #317 to check there

Comment (creisle) [reply docs/inputs/sv_callers.md]: As discussed, seems like there are newer commits in master not tagged so I think we will leave this version blank for the purposes of this PR

**Review by mattdoug604 — APPROVED** [review]

**Pull request merged**